docs/settings/security-settings.asciidoc

```diff
@@ -26,6 +26,112 @@ screens, and authorization using <<kibana-privileges>>. To disable
 Set to `true` to enable audit logging for security events. By default, it is set
 to `false`. For more details see <<xpack-security-audit-logging>>.
 
+[float]
```

Comment:
__issue:__ same comment as in 7.3 version of this PR https://github.com/elastic/kibana/pull/96473#discussion_r609585716. Unfortunately we didn't support this new config format until 7.7.